rotkehlchen/chain/evm/decoding/types.py

```diff
@@ -45,3 +45,5 @@ class EventCategory(SerializableEnumNameMixin):
     CANCEL_ORDER = auto()
     REFUND = auto()
     FEE = auto()
+    MEV = auto()
```

Comment:
please name this MEV_REWARD. MEV is way too generic. We may have different type of MEV events in the frontend in the future.
```suggestion
    MEV_REWARD = auto()
```

Question. Renaming those is it easy in the future? Is it just a frontend thing? So can easily just rename here and in frontend and we are good?